{"name": "test_auditlog_defaultsele3", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_system.test_auditLog.TestAuditLogPage object at 0x000002A582114C70>\nlogin_to_auditlog = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"981a8d9f0748643606694cc6beb56ec7\")>\n\n    @allure.story(\"设置时间为当月，用户为当前登录者，显示异常\")\n    # @pytest.mark.run(order=1)\n    def test_auditlog_defaultsele3(self, login_to_auditlog):\n        driver = login_to_auditlog  # WebDriver 实例\n        audit = AuditLogPage(driver)  # 用 driver 初始化 AuditLogPage\n        name = '洪奥青'\n        today = datetime.now().date()\n        # 格式化为年/月/日\n        formatted_date = today.strftime(\"%Y/%m/%d\")\n        first_day = datetime(today.year, today.month, 1).date()\n        formatted_first_day = first_day.strftime(\"%Y/%m/%d\")\n        ele = audit.get_find_element_xpath('//div[span[text()=\"操作时间:\"]]//input')\n        ele.send_keys(Keys.CONTROL, \"a\")\n        ele.send_keys(Keys.DELETE)\n        audit.enter_texts('//div[span[text()=\"操作时间:\"]]//input', f\"{formatted_first_day} - {formatted_date}\")\n>       audit.enter_texts('//div[span[text()=\"用户:\"]]//input', name)\n\ntest_auditLog.py:126: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\Pages\\systemPage\\auditLog_page.py:20: in enter_texts\n    self.enter_text(By.XPATH, xpath, text)\n..\\..\\Pages\\base_page.py:66: in enter_text\n    element = WebDriverWait(self.driver, wait_time).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"981a8d9f0748643606694cc6beb56ec7\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000002A5820D0430>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Wait until the method returns a value that is not False.\n    \n        Calls the method provided with the driver as an argument until the\n        return value does not evaluate to ``False``.\n    \n        Parameters:\n        -----------\n        method: callable(WebDriver)\n            - A callable object that takes a WebDriver instance as an argument.\n    \n        message: str\n            - Optional message for :exc:`TimeoutException`\n    \n        Return:\n        -------\n        object: T\n            - The result of the last call to `method`\n    \n        Raises:\n        -------\n        TimeoutException\n            - If 'method' does not return a truthy value within the WebDriverWait\n            object's timeout\n    \n        Example:\n        --------\n        >>> from selenium.webdriver.common.by import By\n        >>> from selenium.webdriver.support.ui import WebDriverWait\n        >>> from selenium.webdriver.support import expected_conditions as EC\n    \n        # Wait until an element is visible on the page\n        >>> wait = WebDriverWait(driver, 10)\n        >>> element = wait.until(EC.visibility_of_element_located((By.ID, \"exampleId\")))\n        >>> print(element.text)\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nD:\\Program Files\\Python310\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:146: TimeoutException"}, "attachments": [{"name": "test_auditlog_defaultsele3_failure", "source": "37d8d7ed-c627-4162-bce0-c8cd838c4be0-attachment.png", "type": "image/png"}, {"name": "test_auditlog_defaultsele3_source", "source": "1370ab1f-53d1-4ccf-bcb9-7a77b87027e6-attachment.html", "type": "text/html"}, {"name": "log", "source": "de58445f-f177-45a6-a562-38e6bd5306fb-attachment.txt", "type": "text/plain"}], "start": 1758014765636, "stop": 1758014777148, "uuid": "016c714f-174b-4fc4-a48c-4c5f268a1acc", "historyId": "c239ec82284a0cd9d618d730c3b4da6a", "testCaseId": "c239ec82284a0cd9d618d730c3b4da6a", "fullName": "Tests.test_system.test_auditLog.TestAuditLogPage#test_auditlog_defaultsele3", "labels": [{"name": "story", "value": "设置时间为当月，用户为当前登录者，显示异常"}, {"name": "feature", "value": "审核日志页用例"}, {"name": "tag", "value": "run(order=209)"}, {"name": "parentSuite", "value": "Tests.test_system"}, {"name": "suite", "value": "test_auditLog"}, {"name": "subSuite", "value": "TestAuditLogPage"}, {"name": "host", "value": "WIN-20240820MJV"}, {"name": "thread", "value": "23296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_system.test_auditLog"}]}